config/config.go

```diff
@@ -66,6 +66,7 @@ const (
 	EnvLoginTokenSigningKey        = "LOGIN_TOKEN_SIGNING_KEY"
 	EnvLoginTokenExpirationSeconds = "LOGIN_TOKEN_EXPIRATION_SECONDS"
 	EnvIstioNamespace              = "ISTIO_NAMESPACE"
+	EnvAuthorizeExternalServices   = "AUTHORIZE_EXTERNAL_SERVICES"
```

Comment:
I think we want to have this as a configuration option for each service.

For instance, I could have just Jeager be behind an oauth proxy but not grafana.

Reply:
+1. I think we should get rid of this top level "authorize_external_service" setting and put one under each grafana and tracing - it makes sense here under external_services since this affects ... external services :)

```
external_services:
  grafana:
    confirm_access: true
  tracing:
    confirm_access: true
```      

I don't understand the use of the word "authorize" here (what does "authorize_external_service" mean?). But you get the idea. Perhaps "confirm_access" or something? Then again, I'm not quite sure what this is doing - sounds like its just making a request to confirm you can access the external service - if so, I think this should be "confirm_access".